core/assets/js/scripts.js

```diff
@@ -63,8 +63,12 @@ jQuery( document ).ready(
 
 		jQuery( ".activate-addon" ).click(
 			function (e) {
-				var PluginSlug = jQuery( this ).attr( 'data-plugin-slug' );
-				jQuery('[data-plugin="'+ PluginSlug +'"] .activate a')[0].click();
+				var PluginSlug = jQuery(this).attr('data-plugin-slug');
+				if (typeof jQuery('[data-plugin="' + PluginSlug + '"] .activate a')[0] !== 'undefined') {
```

Comment:
Let's assign `jQuery('[data-plugin="' + PluginSlug + '"] .activate a')[0]` to a variable here, please. Otherwise jQuery has to do the look-up twice and that is slowing things down.